schema.ts

```diff
@@ -1048,7 +1048,17 @@ export type PropertyIdentifier<K extends string = string> = K | string | Propert
  */
 export type ObjectSchemaPathProperties = Pick<
   GenericObjectSchema,
-  'titleProperty' | 'linkProperty' | 'imageProperty' | 'snippetProperty' | 'subtitleProperties' | 'createdAtProperty' | 'createdByProperty' | 'modifiedAtProperty' | 'modifiedByProperty'
+  | 'titleProperty'
+  | 'linkProperty'
+  | 'imageProperty'
+  | 'snippetProperty'
+  | 'subtitleProperties'
+  | 'createdAtProperty'
+  | 'createdByProperty'
+  | 'modifiedAtProperty'
+  | 'modifiedByProperty'
+  | 'userEmailProperty'
+  | 'userIdProperty'
```

Comment:
FWIW it's nice to take advantage of diffs like this as opportunities to alphabetize without unnecessary thrash